pkg/install/v1alpha1/version_test.go

```diff
@@ -40,3 +41,29 @@ func TestIsCurrentVersionValid(t *testing.T) {
 		})
 	}
 }
+
+func TestVersion(t *testing.T) {
+        tests := map[string]struct {
+                version    string
+                expversion string
+        }{
+                "case1": {
+                        version:    "0.7.0",
+                        expversion: "0.7.0",
+                },
+                "case 2": {
+                        version:    "",
+                        expversion: "invalid.version",
+                },
+        }
+
+        for name, tt := range tests {
+                t.Run(name, func(t *testing.T) {
+                        validatedVersion := Version(string(tt.version))
+                        if !reflect.DeepEqual(string(validatedVersion), tt.expversion) {
+                                t.Errorf("Version error %v", validatedVersion)
+                        }
```

Comment:
@Pensu thanks for the PR ..... can we make error more clearer. for example:
t.Errorf ( "version error, got version: %v , expected version: %v ", validatedVersion, tt.expversion)

Reply:
@prateekpandey14 Made the changes. PTAL.